man/openmp-utils.Rd

```diff
@@ -149,6 +149,8 @@ Apart from increasing the size of the input data, function-specific parameters w
       \item Having more and/or complex conditional logic when using \code{\link{fifelse}()} or \code{\link{subset}()}
       }
 
+\code{data.table} also incorporates throttling mechanisms to prevent the excessive creation of threads (can lead to significant overhead) for small data tasks.
```

Comment:
I don't think this is specific enough to be useful. we already have
```
  \item{throttle}{ 1024 (default) means that, roughly speaking, a single thread will be used when nrow(DT)<=1024, 2 threads when nrow(DT)<=2048, etc. The throttle is to speed up small data tasks (especially when repeated many times) by not incurring the overhead of managing multiple threads. Hence the number of threads is throttled (restricted) for small tasks. }
```

Reply:
True